tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -29,14 +30,18 @@ def restore(self, keys) -> dict:
         Fetch all under the same parition key(keys).
         ret: <dict of key to states>
         """
+        tranlated_items = {}
         try:
-            items = zip(
-                keys,
-                (self[key] for key in keys),
-            )
+            first_items = self._get_first_items(keys)
+            remaining_items = self._get_remaining_items(first_items)
+            items = self._merge_items(first_items, remaining_items)
+            #TODO: remove this after berkleyDB is removed.
+            for key in keys:
```

Comment:
Why do we need this? Are the keys passed in not strings?

Reply:
No. Keys passed in are ShelveKeys used by shelvestore. I use them in mirror_state_store. I'll change everything to string once we get rid of shelvestore.